doc/classes/Timer.xml

```diff
@@ -15,20 +15,21 @@
 			<return type="float">
 			</return>
 			<description>
-				Return the time left for timeout in seconds if the timer is active, 0 otherwise.
+				Returns the time left for timeout in seconds if the timer is active, 0 otherwise.
 			</description>
 		</method>
 		<method name="is_paused" qualifiers="const">
 			<return type="bool">
 			</return>
 			<description>
-				Return if the timer is paused or not.
+				If [code]true[/code], the timer is paused.
```

Comment:
Remove comma.

"If true" is for members. This is a function, so "Returns..." is still the correct construction.

Returns [code]true[/code] if the timer is paused.